webapp/client/src/components/StepNavButtons.js

```diff
@@ -60,6 +60,44 @@ const Link = styled.a`
   }
 `;
 
+const OutlineLink = styled.a`
+  ${sharedButtonLinkStyle}
+
+  padding: 1rem 1.25rem calc(1rem - 4px) 1.25rem; /* The calculation subtracts the border-bottom height. We need a border-bottom for the focus state. */
+
+  font-weight: var(--font-bold);
+  letter-spacing: var(--tracking-wide);
+  text-decoration: none;
+  color: var(--link-color);
+
+  background: white;
```

Comment:
There is a variable --bg-white for that (although it wouldn't be necessary to have vars for black and white I think)

Reply:
Thanks! :+1: :grin: